lms/static/scripts/frontend_apps/components/dashboard/DashboardApp.tsx

```diff
@@ -16,6 +17,8 @@ import OrganizationActivity from './OrganizationActivity';
 export default function DashboardApp() {
   const { dashboard } = useConfig(['dashboard']);
 
+  usePlaceholderDocumentTitle();
```

Comment:
Maybe rename this to `usePlaceholderDocumentTitleInDev` or similar to be super explicit that this has no effect in production.